azure-pipelines.yml

```diff
@@ -269,6 +250,25 @@ stages:
       - task: NuGetToolInstaller@1
         displayName: "Install NuGet"
 
+      - task: UseDotNet@2
```

Comment:
And in commit 7442d39c01f687f5ef02bfa706cbc I added `UseDotnet` under the job that was running the UTs, and now I add under the job that is running the ITs (they run in parallel)